ex3/ex3.c

```diff
@@ -9,6 +9,17 @@
 int main(int argc, char* argv[])
 {
     // Your code here
+    int rc = fork();
+
+    if (rc < 0) { // Fail -> exit
+        fprintf(stderr, "fork failed\n");
+        exit(1);
+      } else if (rc == 0) { // Child process satisfying branch
+        printf("Hello\n");
+      } else { // Parent process
+        int wc = waitpid(rc, NULL, 0);
```

Comment:
When compiling with the `-Wall` and `-Wextra` arguments (error reporting), this line throws the warning `implicit declaration of function 'waitpid' [-Wimplicit-function-declaration]` - this won't prevent the code from compiling, however it can lead to issues as implicit declarations results in functions that can't be properly checked to verify they're being used correctly, which can lead to unexpected behavior. This variable also isn't being used, though that's hardly an issue. Good work overall in this exercise!